.github/workflows/e2e-tests.yml

```diff
@@ -98,12 +98,16 @@ jobs:
     - name: Run all tests
       if: matrix.gutenberg == 'no'
       working-directory: projects/plugins/jetpack/tests/e2e
-      run: yarn test-e2e
+      run: yarn test-e2e --testPathIgnorePatterns=search
 
     - name: Run only blocks tests with latest Gutenberg
       if: matrix.gutenberg == 'latest'
       working-directory: projects/plugins/jetpack/tests/e2e
       run: yarn test-e2e --testNamePattern=blocks
+    - name: Run only search tests
+      if: matrix.gutenberg == 'no' && ( contains(github.event.pull_request.labels.*.name, 'Search') || contains(github.event.pull_request.labels.*.name, 'Instant Search') )
```

Comment:
I'm not totally against this complex conditional logic, but I wonder what problem are we trying to solve here?

Reply:
@brbrr because Jetpack Instant Search is essentially a separate application, I don't think we need to run the Search test suite unless the PR is related to search (as indicated by those labels). The aim here is to avoid adding several minutes of tests to the main suite.

Reply:
explained to @brbrr on Slack